test/test-main.js

```diff
@@ -0,0 +1,10 @@
+global.DATABASE_URL = 'mongodb://localhost/basic-auth-test'
+global.PORT = '8081'
```

Comment:
Let's use [this approach](https://github.com/Thinkful-Ed/node-restaurants-app-mongoose/blob/feature/with-tests/server.js#L133) for DATABASE_URL and PORT in runserver so we can avoid globals.